packages/aws-cdk-lib/aws-elasticloadbalancingv2/test/nlb/target-group.test.ts

```diff
@@ -144,6 +144,25 @@ describe('tests', () => {
     }).toThrow(/Health check interval '3' not supported. Must be between 5 and 300./);
   });
 
+  test('Throws error for health check interval less than timeout', () => {
```

Comment:
Could you add an edge case test for `interval == timeout`?

Reply:
I added a test for this. See [discussion above](https://github.com/aws/aws-cdk/pull/29075#discussion_r1501803750) on why we chose to allow them to be equal.